src/cli.ts

```diff
@@ -32,14 +41,19 @@ export function run(): Promise<void> {
     }
 
     if (cliOptions.help) {
-      var usage = cli.getUsage({
-        header: 'A development server for Polymer projects',
-        title: 'polyserve',
-      });
-      console.log(usage);
+      printUsage(cli);
+      reject();
```

Comment:
Hey, okay my line of thought was that the function should only resolve when it "runs" the server, but resolving with null could also be used to signal no server was started.